src/css/profile/mobile/common/listview.less

```diff
@@ -61,12 +61,64 @@ tau-expandable {
 		overflow: visible;
 		text-align: left;
 		font-size: @font_size_list_main_text;
-		min-height: 50 * @px_base;
 
 		&[disabled] {
 			opacity: 0.4;
 		}
 
+		// Subheader
+		&.ui-li-subheader {
+			margin-left: 20 * @px_base;
+			flex-direction: row;
+			align-items: center;
+
+			&:empty {
+				margin-top: 16 * @px_base;
+			}
+
+			.ui-li-text-subheader {
+				display: inline-flex;
+				color: var(--text-secondary-color);
+				font-family: Roboto-Medium;
```

Comment:
For family font we have `less` method `.font(medium)`, `.font(regular)`, `.font(light)`.

Reply:
You can check this method in tizen.less

Reply:
Good Tip! I will check it.